<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sqform</class>
 <widget class="QWidget" name="Sqform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Введите действительное число, коэффициент при x^2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="enterAX"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Введите действительное число, коэффициент при x</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="enterBX"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Введите действительное число, свободный член</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="enterCX"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="enterButton">
     <property name="text">
      <string>Ввод</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Решение уравнения:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ans">
       <property name="text">
        <string>Ответ:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
